caddyhttp/staticfiles/fileserver.go

```diff
@@ -40,6 +40,14 @@ func (fs FileServer) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, err
 	return fs.serveFile(w, r, r.URL.Path)
 }
 
+// calculateEtag produces a strong etag by default. Prefix the result with "W/" to convert this into a weak one.
+// see https://tools.ietf.org/html/rfc7232#section-2.3
+func calculateEtag(d os.FileInfo) string {
+	t := strconv.FormatInt(d.ModTime().Unix(), 36)
+	s := strconv.FormatInt(d.Size(), 36)
+	return fmt.Sprintf(`"%s%s"`, t, s)
+}
```

Comment:
Yes, well spotted, it would be (and I asked myself the same question). But the double quote marks are required (note these are enclosed in the back-ticks). It's a feature of ETags that there need to be double-quote marks. A bit unnecessary I agree, but that's what the spec requires.